### PR TITLE
Add script for setting up farms. Update Python tool.

### PR DESCRIPTION
**Pull request by deuszx**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,8 +13,9 @@ addresses.json
 
 # Ignore Rust contract wrappers.
 amm/drink-tests/src/*_contract.rs
+amm/drink-tests/src/psp22.rs
+amm/drink-tests/src/wrapped_azero.rs
```
Comment (h4nsu) [inline .gitignore:17]: If you're adding these to `.gitignore`, please delete corresponding files from the PR

File: farm/tests/src/psp22/psp22_contract.rs [hunk 2]
```diff
@@ -0,0 +1,267 @@
+use scale::Encode as _;
```
Comment (h4nsu) [inline farm/tests/src/psp22/psp22_contract.rs:1]: This file is auto-generated. Shouldn't it be in `.gitignore`?

File: scripts/commonfarms-cli/commonfarms-cli [hunk 3]
```diff
@@ -13,18 +13,23 @@ def argument_parser():
     parser = ArgumentParser(prog='commonfarms-cli', description='Command line tool for management of Common Farms')
     parser.add_argument('--chain', metavar='URL', default='local', help='WebSocker URL of the chain (possible shortcuts: mainnet, testnet, local)')
     parser.add_argument('--phrase', metavar='SEED', help='seed phrase of the farm admin account (if not supplied, an interactive prompt will ask for it)')
-    parser.add_argument('--metadata', metavar='PATH', default='../../artifacts/farm_contract.json', help='path to farm contract metadata file')
+    parser.add_argument('--farm-metadata', metavar='PATH', default='../../artifacts/farm_contract.json', help='path to farm contract metadata file')
+    parser.add_argument('--psp22-metadata', metavar='PATH', default='../../artifacts/psp22.json', help='path to PSP22 contract metadata file')
```
Comment (h4nsu) [inline scripts/commonfarms-cli/commonfarms-cli:17]: We could consider dropping these two arguments and fetching them from `artifacts`

**Review by h4nsu — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (deuszx) [reply .gitignore:17]: Ah, I will keep them committed into the repo. I'll remove from gitignore. I had problems with `CodeHashMismatch` but it was due to the fact that I had incompatible `*.contract` and `*.json` files.

File: farm/tests/src/psp22/psp22_contract.rs (hunk 2 above)
Comment (deuszx) [reply farm/tests/src/psp22/psp22_contract.rs:1]: It's auto-generated but the contracts used for generating it are hardcoded into the project (`psp22` and `wrapped_azero`) so it doesn't need to be re-generated.

File: scripts/commonfarms-cli/commonfarms-cli (hunk 3 above)
Comment (deuszx) [reply scripts/commonfarms-cli/commonfarms-cli:17]: I can imagine someone just "copying" the file w/o arficats. Let's leave as-is, defaults are good.

File: .gitignore (hunk 1 above)
Comment (h4nsu) [reply .gitignore:17]: If this is the case then both these files contain strings with paths from your local machine. Please fix this

File: amm/drink-tests/src/psp22.rs [hunk 4]
```diff
@@ -243,7 +243,9 @@ impl PSP22 for Instance {
 }
 #[allow(dead_code)]
 pub fn upload() -> ink_wrapper_types::UploadCall {
-    let wasm = include_bytes!("../../../artifacts/psp22.wasm");
+    let wasm = include_bytes!(
+        "/Users/gorskimateusz/projects/aleph/dex/amm/scripts/../../artifacts/psp22.wasm"
```
[the remark this answers is not in the document]
Comment (deuszx) [reply amm/drink-tests/src/psp22.rs:247]: oh... missed that

[the remark this answers is not in the document]
Comment (deuszx) [reply amm/drink-tests/src/psp22.rs]: done

File: amm/drink-tests/src/wrapped_azero.rs [hunk 5]
```diff
@@ -276,9 +207,80 @@ impl PSP22 for Instance {
...
+    ) -> ink_wrapper_types::ReadCall<Result<u8, ink_wrapper_types::InkLangError>> {
+        let data = vec![114u8, 113u8, 183u8, 130u8];
+        ink_wrapper_types::ReadCall::new(self.account_id, data)
+    }
+}
 #[allow(dead_code)]
 pub fn upload() -> ink_wrapper_types::UploadCall {
-    let wasm = include_bytes!("../../../artifacts/wrapped_azero.wasm");
+    let wasm = include_bytes!(
+        "/Users/gorskimateusz/projects/aleph/dex/amm/scripts/../../artifacts/wrapped_azero.wasm"
```
[the remark this answers is not in the document]
Comment (deuszx) [reply amm/drink-tests/src/wrapped_azero.rs]: done

**Review by h4nsu — APPROVED** [review]

**Pull request merged**